Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:20.04 as build
+ENV TZ=Etc/UTC
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+# Install dependencies
+## apt install python & libraries
+RUN apt-get update && \
+    apt-get install -y \
+        python3 \
+        python3-pip \
+        libjpeg-dev \
+        zlib1g-dev \
+        libsdl2-dev \
+        git \
+        wget
+
+## install python dependencies
+COPY requirements.txt /tmp/
+RUN pip3 install -r /tmp/requirements.txt
+
+## Windows build dependencies
+#RUN apt-get install -y \
+#        binutils-mingw-w64
+#RUN apt-get install -y \
+#        tcc \
+#        unzip
+
+## Switch build dependencies
```

Comment:
Why not break out to separate `Dockerfile`s per target?

Reply:
That's not a bad idea. My original thought was to specify targets using arguments. But different Dockerfiles per target would be clearer and split out the process to be maintained separately.

When I have some free cycles, I'll redo this PR. Unless someone else beats me to it 😉😂

Reply:
I'm not a docker expert, but I think duplicating information into multiple Dockerfiles is not very DRY.

Is there a way to have a common dockerfile for the others to reference?

Reply:
I think it is better to keep it in one Dockerfile, that way you can call both jobs, reusing code and decreasing build time.